frontend/views/components/group-creation-steps/GroupRules.vue

```diff
@@ -96,13 +37,36 @@ export default {
     $v: { type: Object }
   },
   components: {
-    CircleSlider,
-    BannerSimple
-  },
-  filters: {
-    toPercent
+    TransitionExpand,
+    VotingSystemInput
   },
+  data: () => ({
+    config: {
+      [RULE_PERCENTAGE]: {
+        optionLabel: L('Percentage based'),
+        optionHint: L('Define the percentage of members who will need to agree to a proposal.', LTags('b')),
+        slideDefault: 75 // TODO connect to store.
+      },
+      [RULE_DISAGREEMENT]: {
+        optionLabel: L('Disagreement number'),
+        optionHint: L('Define the number of people required to block a proposal.', LTags('b')),
```

Comment:
Here and above `{b}` is not used, so shouldn't be included.

Reply:
Removed